consoleme/handlers/v2/roles.py

```diff
@@ -335,6 +335,20 @@ async def get(self, account_id, role_name):
             role_details = None
             error = str(e)
 
+        allowed = True
+
+        allowed_tags = config.get("roles.allowed_tags", {})
+        actual_tags = {k: v for d in role_details.tags for k, v in d.items()} # Convert List[Dicts] to 1 Dict
```

Comment:
`role_details` may be `None` here, in the case that role doesn't exist! (line 352 check)

Reply:
Oof, yea, let me fix that